src/fi_tostr.c

```diff
@@ -883,6 +1043,18 @@ char *DEFAULT_SYMVER_PRE(fi_tostr_r)(char *buf, size_t len,
 	case FI_TYPE_LOG_SUBSYS:
 		ofi_tostr_log_subsys(buf, len, *enumval);
 		break;
+	case FI_TYPE_AV_ATTR:
+		ofi_tostr_av_attr(buf, len, data, "");
+		break;
+	case FI_TYPE_CQ_ATTR:
+		ofi_tostr_cq_attr(buf, len, data, "");
+		break;
+	case FI_TYPE_MR_ATTR:
+		ofi_tostr_mr_attr(buf, len, data, "");
+		break;
+	case FI_TYPE_CNTR_ATTR:
+		ofi_tostr_cntr_attr(buf, len, data, "");
+		break;
```

Comment:
We shouldn't need a prefix parameter for the new functions.  We have a prefix for the other attributes because they can be printed separately or as part of printing fi_info.